File: Neetcode 150/Array s-Hashing/two sum/twoSum.ts
function twoSum(nums: number[], target: number): number[] {
  const hashIdx = {};
  for (let i = 0; i < nums.length; i++) {
    const sub = target - nums[i];
    if (sub in hashIdx) {
      return [i, hashIdx[sub]];
    } else {
      hashIdx[nums[i]] = i;
    }
  }
}

File: Neetcode 150/Array s-Hashing/two sum/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashIdx = {}
        for i in range(len(nums)):
            sub = target - nums[i]
            if sub in hashIdx:
                return [i, hashIdx[sub]]
            else:
                hashIdx[nums[i]] = i
        
        
        